### DISCUSSION TITLE
PHP Deprecated:  htmlspecialchars(): Passing null to parameter

### DISCUSSION THREAD

**Discussion by davood-cm** [spatie/array-to-xml#186] [Q&A]
#1 ($string) of type string is deprecated in vendor/spatie/array-to-xml/src/ArrayToXml.php on line 144

version: 3.1.2
php: 8.1.4

**Comment by freekmurze** [comment]
Feel free to send a PR that fixes this.

**Comment by gigerIT** [comment]
If you're on it there is another one 
`DOMDocument::__construct(): Passing null to parameter #2 ($encoding) of type string is deprecated in vendor/spatie/array-to-xml/src/ArrayToXml.php on line 29`
Deprecated in PHP 8.1
